src/udev/udev-manager.c

```diff
@@ -857,6 +857,58 @@ static void manager_set_default_children_max(Manager *manager) {
         log_debug("Set children_max to %u", manager->children_max);
 }
 
+static int split_assignment(const char *assignment, char **ret_key, char **ret_value) {
```

Comment:
Oh, you already fixed the issue. Please merge this commit with the previous one.

Reply:
No, this still inserts a NULL value into the hashmap when "removing" a property. As said, I thought that was intentional behavior.